exercises/C-comparison-operators/exercise.js

```diff
@@ -7,18 +7,19 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
+
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom =  roomMaxCapacity >= // finish this statement
```

Comment:
Hi Pamela, is this code in line 14 intentional? I mean are you still working on it? If not, you just need to complete it and it will pass the test. So you need to add : studentCount; I hope this helps.